pkg/controller/registry/reconciler/grpc.go

```diff
@@ -132,15 +133,15 @@ func (c *GrpcRegistryReconciler) EnsureRegistryServer(catalogSource *v1alpha1.Ca
 
 	// if service status is nil, we force create every object to ensure they're created the first time
 	overwrite := source.Status.RegistryServiceStatus == nil
-	// recreate the pod if no existing pod is serving the latest image
+	// recreate the pod if no existing pod is serving the latest imageID
 	overwritePod := overwrite || len(c.currentPodsWithCorrectImage(source)) == 0
 
 	//TODO: if any of these error out, we should write a status back (possibly set RegistryServiceStatus to nil so they get recreated)
 	if err := c.ensurePod(source, overwritePod); err != nil {
 		return errors.Wrapf(err, "error ensuring pod: %s", source.Pod().GetName())
 	}
 	if err := c.ensureUpdatePod(source); err != nil {
-		return errors.Wrapf(err, "error ensuring updated catalog source pod: %s", source.Pod().GetName())
+		return errors.Wrapf(err, "error ensuring imageChanged catalog source pod: %s", source.Pod().GetName())
```

Comment:
This looks like an unintended find/replace?

Reply:
yes, fixed